Week1-step3/Week1-step3/main.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  main.swift
+//  Week1-step3
+//
+//  Created by 민경환 on 2022/06/30.
+//
+
+import Foundation
+
+func makerChocoStick(길이: Int, 몸통: String, 토핑: String, 막대길이: Int) {
```

Comment:
```suggestion
func makeChocoStick(길이: Int, 몸통: String, 토핑: String, 막대길이: Int) {
```

확실한 동사로 표현해주는게 좋을 것 같습니다!